src/compiler/checker.ts

```diff
@@ -2835,17 +2876,25 @@ module ts {
             return result;
         }
 
-        function getExportsOfExternalModule(node: ImportDeclaration): Symbol[]{
+        function symbolsToArray(symbols: SymbolTable): Symbol[] {
+            let result: Symbol[] = [];
+            for (let id in symbols) {
+                if (!isReservedMemberName(id)) {

```

Comment:
We don't want the reserved member names (such as '__export') to show up in statement completion lists.